packages/amplify_datastore/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -693,14 +693,15 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
```

Comment:
Are the DEVELOPMENT_TEAM and PRODUCT_BUNDLE_IDENTIFIER changes here intentional?

Reply:
Unintentional, thanks.